api/emoji.go

```diff
@@ -122,6 +122,10 @@ func createEmoji(c *Context, w http.ResponseWriter, r *http.Request) {
 		c.Err = result.Err
 		return
 	} else {
+		message := model.NewWebSocketEvent(model.WEBSOCKET_EVENT_EMOJI_ADDED, "", "", "", nil)
+		message.Add("emoji", result.Data.(*model.Emoji).ToJson())
+
+		go app.Publish(message)
```

Comment:
Sorry, totally missed these messages. The problem with creating unbounded goroutines like this is that they are essentially resource leaks once the system can no longer keep and stays under a large amount of load. What could happen for example (and we've seen this happen) is that the channels on the hubs back up due to too much load and then number of goroutines continuously rises until the server runs out of memory and crashes. So it's better to block on the Publish in the case where there's high load, rather than resource leak goroutines and possibly cause a server crash.

I know we do use `go Publish` in many places but that's from legacy code before we realized the potential dangers of using unbounded goroutines so excessively